.github/workflows/test_with_clone.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
                 snow sql -q "USE ROLE SECURITYADMIN; GRANT OWNERSHIP ON FUTURE DYNAMIC TABLES IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE TO ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE};"
                 snow sql -q "USE ROLE SECURITYADMIN; GRANT OWNERSHIP ON FUTURE TASKS IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE_RAW TO ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE} COPY CURRENT GRANTS;"
```

Comment:
Add one more query for `synapse_aggregate` schema here

Reply:
There aren't any tasks or dynamic tables in SYNAPSE_AGGREGATE. Would it be premature to add a line here?

Reply:
No, since the logic in the PR for `synapse_aggregate` implies we will be introducing dynamic tables we should prepare for that in the CI/CD. We _could_ opt-out the FUTURE TASKS query, though, if you're not planning to have tasks in the gold-table schemas.

Reply:
Makes sense. I've added the line for DYNAMIC TABLES.

Reply:
Whoops, I originally misread this. I added two lines: one for granting ownership of all dynamic tables to the clone proxy admin (even though we don't yet have any dynamic tables) and another for revoking ownership of future dynamic tables from the `<DOMAIN>_PROXY_ADMIN`

Reply:
lgtm!